Source/WPEFramework/PluginServer.cpp

```diff
@@ -701,7 +702,10 @@ namespace PluginHost
                 result = Core::ERROR_NONE;
                 #endif
                 if (result == Core::ERROR_NONE) {
+                    const string className(PluginHost::Service::Configuration().ClassName.Value());
```

Comment:
As you are now in the class Service (which inherits from Plugin::Service) There is no need to dive in the config. The service already has convenience methods to return Classname and Callsign, see: https://github.com/rdkcentral/Thunder/blob/master/Source/plugins/Service.h#L192-L199
As potentially this might change in the future, suggest not to copy it;s content but just use these convenience methods.. So you can just code:
`SYSLOG(Logging::Notification, ("Hibernated plugin [%s]:[%s]", ClassName().c_str(), Callsign().c_str()))`
and drop all the temporary variables.

Reply:
gotcha, changed per your suggestion